lib/app/features/feed/create_article/providers/create_article_notifier.c.dart

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:ion/app/features/feed/data/models/entities/article_data.c.dart';
+import 'package:ion/app/features/feed/views/components/text_editor/components/custom_blocks/text_editor_single_image_block/text_editor_single_image_block.dart';
+import 'package:ion/app/features/gallery/providers/providers.dart';
+import 'package:ion/app/features/nostr/model/file_alt.dart';
+import 'package:ion/app/features/nostr/model/file_metadata.c.dart';
+import 'package:ion/app/features/nostr/model/media_attachment.dart';
+import 'package:ion/app/features/nostr/providers/nostr_notifier.c.dart';
+import 'package:ion/app/features/nostr/providers/nostr_upload_notifier.c.dart';
+import 'package:ion/app/services/compressor/compress_service.c.dart';
+import 'package:ion/app/services/media_service/media_service.c.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'create_article_notifier.c.g.dart';
+
+@Riverpod(dependencies: [])
+class CreateArticleNotifier extends _$CreateArticleNotifier {
```

Comment:
I think we don't need to add a suffix related to the provider for providers; we can just use `CreateArticle`. wdyt?